### ISSUE TITLE
Notice:  Undefined index: allowed_html in [server_path]/wp-content/plugins/redux-framework/ReduxCore/inc/validation/html_custom/validation_html_custom.php on line 30

### ISSUE THREAD

**Issue by MB-Jan** [reduxframework/redux-framework#2893]
Hi there,

This is my first post on GitHub, I hope I follow the rules. Clicking on "Save Changes" resulted in a javascript alert: "There was a problem with your action. Please try again or reload the page."

In the console I got this information:
"Notice:  Undefined index: allowed_html in [server_path]/wp-content/plugins/redux-framework/ReduxCore/inc/validation/html_custom/validation_html_custom.php on line 30"

Even though the error popped up, the changes were successfully saved. I then modified the validation_html_custom.php file to check if the index exists. Here is how the file currently looks like:
`<?php

    if ( ! class_exists( 'Redux_Validation_html_custom' ) ) {
        class Redux_Validation_html_custom {

            /**
             * Field Constructor.
             * Required - must call the parent constructor, then assign field and value to vars, and obviously call the render field function
             *
             * @since ReduxFramework 1.0.0
             */
            function __construct( $parent, $field, $value, $current ) {

                $this->parent  = $parent;
                $this->field   = $field;
                $this->value   = $value;
                $this->current = $current;

                $this->validate();
            } //function

            /**
             * Field Render Function.
             * Takes the vars and validates them
             *
             * @since ReduxFramework 1.0.0
             */
            function validate() {
                if(array_key_exists('allowed_html', $this->field)) {
                       $this->value = wp_kses( $this->value, $this->field['allowed_html'] );
                }
            } //function
        } //class
    }`

This fixes the issue. I would appreciate if you could add this to the plugin, so it will stay updatable. 
Thank you.

Best regards,

Jan

**Comment by kprovance** [comment]
While feasible, I need to know why it's happening, that is, which part of your options array is making that happen.  Do you know?

**Comment by MB-Jan** [comment]
Hi kprovance,

Thank you for your swift reply. So I removed all theme options and added one options array by one. I realized that the option setup itself is not the issue. With empty fields the error is not happening. The error seems to be caused by the Thai language used in text and editor fields. As soon as Thai text is present, the error/alert window pops up, when we like to save the changes.

I am not really comfortable to share the whole json output from the console here in public, as it contains some (semi-)private information. If required, is there any way I could send you the output privately?

Thank you.
